std-bits/table/src/main/java/org/enso/table/data/column/builder/DoubleBuilder.java

```diff
@@ -16,7 +16,7 @@
 import org.enso.table.util.BitSets;
 
 /** A builder for floating point columns. */
-class DoubleBuilder extends NumericBuilder implements BuilderForDouble {
+non-sealed class DoubleBuilder extends NumericBuilder implements BuilderForDouble {
```

Comment:
Why not 
```suggestion
sealed class DoubleBuilder extends NumericBuilder implements BuilderForDouble permits
    InferredDoubleBuilder {
```
The `InferredDoubleBuilder is in the same package